src/UiSlider.vue

```diff
@@ -316,9 +303,13 @@ export default {
             this.dragUpdate(e);
         },
 
+        getTrackLength() {
+            return this.$refs.track.offsetWidth;
+        },
+
         dragUpdate(e) {
             const position = e.touches ? e.touches[0].pageX : e.pageX;
-            const relativeValue = (position - this.trackOffset) / this.trackLength;
+            const relativeValue = (position - this.getTrackOffset(this.$refs.track)) / this.getTrackLength();
```

Comment:
`this.getTrackOffset()` doesn't take a parameter:

```suggestion
            const relativeValue = (position - this.getTrackOffset()) / this.getTrackLength();
```

Reply:
Still relevant